app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt

```diff
@@ -198,7 +198,13 @@ class EventDetailsFragment : Fragment() {
     override fun onDestroyView() {
         val activity = activity as? MainActivity
         activity?.supportActionBar?.setDisplayHomeAsUpEnabled(false)
-        activity?.supportActionBar?.title = resources.getString(R.string.events)
+
+        val currentFragment = fragmentManager?.findFragmentById(R.id.frameContainer)
+        if(currentFragment == EventsFragment())
+            activity?.supportActionBar?.title = resources.getString(R.string.events)
+        else
+            activity?.supportActionBar?.title = resources.getString(R.string.likes)
```

Comment:
Wrong logic. This is a workaround, not a fix

Reply:
Should I change it to another if statement instead of else since it will be better so that the Likes case can be handled? Or this logic is completely unacceptable?

Reply:
This should be checked in EventDetailFragment that it is EventsFragment or not. The title changing logic should remain inside its own fragment

Reply:
But the title changing logic was already inside the EventDetailsFragment (For changing the title to 'Events'). Should I modify it to shift it to the respective fragments?

Reply:
Yes